diff --git a/contrib/julia-config.jl b/contrib/julia-config.jl
index 55996e9fe5..8725b08eba 100755
--- a/contrib/julia-config.jl
+++ b/contrib/julia-config.jl
@@ -67,8 +67,9 @@ function ldlibs(doframework)
         "julia"
     end
     if Sys.isunix()
-        return "-L$(shell_escape(private_libDir())) " *  "-Wl,-rpath,$(shell_escape(libDir())) " *
-            (Sys.isapple() ? string() : "-Wl,-rpath,$(shell_escape(private_libDir())) ") *
+        return "-L$(shell_escape(private_libDir())) " *
+            "-Wl,-rpath,$(shell_escape(libDir())) " *
+            "-Wl,-rpath,$(shell_escape(private_libDir())) " *
             "-l$libname"
     else
         return "-l$libname -lopenlibm"
